examples/dimension_expander.rs

```diff
@@ -99,8 +100,8 @@ impl Plugin for DimensionExpander {
         Info {
             name: "Dimension Expander".to_string(),
             vendor: "overdrivenpotato".to_string(),
-            unique_id: 243723071,
-            version: 0001,
+            unique_id: 243_723_071,
```

Comment:
Should these be underscored even though they're IDs and not ints?  Is there any specific reason for `0001` that we should preserve?

Reply:
0001 is numerically the same as 1, if we want to have proper versioning then we need to use a better data type. We should switch to using this https://github.com/steveklabnik/semver

Reply:
we will do this in a later request

Reply:
and maybe we should add a UUID generator to handle the id numbers

Reply:
https://doc.rust-lang.org/uuid/uuid/index.html

Reply:
Is this something specific to the VST spec, though?  Or can any arbitrary datatype be inserted here with a change?  I don't really wish to add another dependency just for versioning - an int is fine for me.

Reply:
imo this adds confusion, using the segmented int stuff. UUID generator would add a dependency which is something we definitely want to avoid. we could just write a simple randomised function perhaps?

Reply:
I will roll this bit back tomorrow if you guys don’t take care of it before then

Reply:
I kinda feel like the example source should be mostly deterministic